debian_on_termux.sh

```diff
@@ -30,8 +30,8 @@ fallback() {
 	cd debootstrap
 }
 
-USER_ID=$(id -u)
-USER_NAME=$(id -un)
+USER_ID=`id -u`
```

Comment:
This is contrary to shellcheck and my preference (use characters on a keyboard where possible)
Please remove this commit from the PR.